migration/v4/model/businessservice.go

```diff
@@ -0,0 +1,10 @@
+package model
+
+type BusinessService struct {
```

Comment:
hint: necessary to bring this forward since it has a relationship to a model that was updated in this migration (Application)